cartography/driftdetect/cli.py

```diff
@@ -30,7 +30,7 @@ def _build_parser(self):
                 'graph database and reports the deviations.'
             ),
             epilog='For more documentation please visit: '
-                   'https://github.com/lyft/cartography/blob/master/docs/drift-detect.md',
+                   'https://lyft.github.io/cartography/usage/drift-detect.html',
```

Comment:
Apologies for the previous change request, I should have checked with Alex first!

Please point to 
https://github.com/lyft/cartography/blob/master/docs/root/usage/drift-detect.md

Reply:
It's good right now